src/js/components/Navigation/MainHeaderBar.jsx

```diff
@@ -75,7 +75,7 @@ const useStyles = makeStyles((theme) => ({
   outerWrapper: {
     borderBottom: '1px solid #ddd',
     flexGrow: 1,
-    minHeight: 42,
+    minHeight: 36,
```

Comment:
Hi @SailingSteve, great work on this pull request. In my next pull request, I will be bringing the minHeight here back to 42 (as opposed to your change to 36) to solve some spacing problems introduced in this pull request. I'll also be using 42px in HeaderBarWrapper and WeVoteLogoWrapper in /components/Navigation/HeaderBarLogo.jsx. Please let me know if you would like to see these changes in a screen share or discuss.